src/test/java/org/assertj/core/util/introspection/PropertyOrFieldSupport_getValueOf_Test.java

```diff
@@ -151,6 +151,27 @@ public void should_extract_single_value_from_maps_by_key() {
     assertThat(maps).extracting("bad key").containsExactly(null, null);
   }
 
+  @Test
+  public void should_extract_field_value_if_only_static_getter_matches_name() {
+    Object value = propertyOrFieldSupport.getValueOf("city", new StaticPropertyEmployee());
+
+    assertThat(value).isEqualTo("New York");
+  }
+
+  @Test
+  public void should_extract_field_value_if_only_static_is_method_matches_name() {
+    Object value = propertyOrFieldSupport.getValueOf("tall", new StaticBooleanPropertyEmployee());
+
+    assertThat(value).isEqualTo(false);
```

Comment:
use `isFalse()`

Reply:
I don't think I can as `value` is an `Object` and there's no `isFalse()` method available on `ObjectAssert`.

Reply:
yes, that's true